flow360/component/flow360_params/flow360_output.py

```diff
@@ -85,24 +137,25 @@ class SurfaceOutput(Flow360BaseModel):
     """:class:`SurfaceOutput` class"""
 
     output_format: Optional[OutputFormat] = pd.Field(alias="outputFormat")
-    animation_frequency: Optional[Union[PositiveInt, Literal[-1]]] = pd.Field(
-        alias="animationFrequency"
-    )
-    animation_frequency_offset: Optional[int] = pd.Field(alias="animationFrequencyOffset")
-    animation_frequency_time_average: Optional[Union[PositiveInt, Literal[-1]]] = pd.Field(
-        alias="animationFrequencyTimeAverage"
-    )
-    animation_frequency_time_average_offset: Optional[int] = pd.Field(
-        alias="animationFrequencyTimeAverageOffset"
-    )
+    animation: Optional[AnimationSettings] = pd.Field(alias="animation")
```

Comment:
1. we also need to have legacy way of defining animations. We can put it in classes SurfaceLegacy (not sure what is the best naming convention here). 

2. We need `to_solver()` function:
```python
class Surface:
   def to_solver(params: Flow360Params) -> SurfaceLegacy:
      pass
```

Reply:
Readded legacy animations along with to_solver methods. Once we switch to the animation_settings structure on the solver side this duplication can be removed and resolved in the updater for back-compatibility